src/redemptions/dto/redemption.dto.ts

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { IsNotEmpty, IsString } from 'class-validator';
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+export class RedemptionDto {
```

Comment:
In other dto's this is named after the action and then the noun, so it should be `CreateRedemptionDto`

Reply:
fixed